diff --git a/node_modules/@react-navigation/core/lib/module/useNavigationBuilder.js b/node_modules/@react-navigation/core/lib/module/useNavigationBuilder.js
index 586b8be..c40528a 100644
--- a/node_modules/@react-navigation/core/lib/module/useNavigationBuilder.js
+++ b/node_modules/@react-navigation/core/lib/module/useNavigationBuilder.js
@@ -1,6 +1,6 @@
 "use strict";
 
-import { CommonActions } from '@react-navigation/routers';
+import { CommonActions, getStateFromPath, validatePathConfig } from '@react-navigation/routers';
 import * as React from 'react';
 import { isValidElementType } from 'react-is';
 import useLatestCallback from 'use-latest-callback';
@@ -30,6 +30,7 @@ import { useOnGetState } from "./useOnGetState.js";
 import { useOnRouteFocus } from "./useOnRouteFocus.js";
 import { useRegisterNavigator } from "./useRegisterNavigator.js";
 import { useScheduleUpdate } from "./useScheduleUpdate.js";
+import { checkDuplicateRouteNames } from "./checkDuplicateRouteNames.js";
 
 // This is to make TypeScript compiler happy
 // eslint-disable-next-line @typescript-eslint/no-unused-expressions
